libs-scala/timer-utils/src/main/scala/com/digitalasset/timer/PromiseTask.scala

```diff
@@ -11,10 +11,10 @@ import scala.util.control.NonFatal
 
 private class PromiseTask[A](value: => Future[A]) extends TimerTask with Promise[A] {
 
-  private val p = Promise[A]
+  private val p = Promise[A]()
 
   override def run(): Unit = {
-    p.tryCompleteWith {
+    p.completeWith {
```

Comment:
OK, clearly this API was already broken, but I'm surprised they deprecated `tryCompleteWith` and not this one, as the semantics are far closer to `trySuccess` and `tryFailure`.

Reply:
:shrug: